sw/ground_segment/python/guided_mode_example.py

```diff
@@ -161,14 +161,26 @@ def shutdown(self):
             self._interface = None
 
     def get_aircrafts(self):
+        wait_step = 0.1
+        timeout = 30 / wait_step  # 30 seconds
+        new_answer = False
 
         def aircrafts_cb(ac_id, msg):
+            global new_answer
             self.ac_list = [int(a) for a in msg['ac_list'].split(',') if a]
             print("aircrafts: {}".format(self.ac_list))
+            new_answer = True
 
         self._interface.send_request('ground', "AIRCRAFTS", aircrafts_cb)
         # hack: sleep briefly to wait for answer
-        sleep(0.1)
```

Comment:
I can remove this commit if you think it's unnecessary or unrelated. I was updating the `send_request` part with the new method and noticed this can be changed a bit.

Reply:
This example should probably use the `pprz_connect` tool, but I guess this is also nice to have an example using the request API directly. So I think we can keep it